### PR TITLE
Add r target

### PR DESCRIPTION
**Pull request by gabrielakoreeda**
Added a r language target using the library httr.

### CODE REVIEW

**Comment by darrenjennings** [comment]
@gabrielakoreeda thank you for this contribution! Our test suite on `master` is now stable. Can you add the output fixtures, fix and linting errors and get the tests passing? `npm test` should guide you in what you need to do. Look to #128 for inspiration.

**Comment by darrenjennings** [comment]
@gabrielakoreeda I see tests are now passing, is this ready to be reviewed?

**Comment by gabrielakoreeda** [comment]
Yes!

File: test/fixtures/output/r/httr/headers.r [hunk 1]
```diff
@@ -0,0 +1,7 @@
+library(httr)
+
+url <- "http://mockbin.com/har"
+
+response <- VERB("GET", url, add_headers(x_foo = 'Bar'), content_type("application/octet-stream"), accept(application/json))
```
Comment (gschier) [inline test/fixtures/output/r/httr/headers.r]: I'm not familiar with R, but should the `application/json` have quotes around it?

Comment (gabrielakoreeda) [reply test/fixtures/output/r/httr/headers.r]: Yes! You're right, I have corrected it now. Thank you!

**Review by gschier — APPROVED** [review]
This looks good to me now. You too @darrenjennings?

**Comment by MarcBT** [comment]
Hi there, thanks for this addition 👍 !
Any chance to have it merged and released soon ?!
Thanks a lot

**Comment by chakpongchung** [comment]
> Hi there, thanks for this addition 👍 !
> Any chance to have it merged and released soon ?!
> Thanks a lot

I need this feature very much now! please merge and release!

**Comment by wilder** [comment]
Nice one, @gabrielakoreeda !

**Comment by chakpongchung** [comment]
how can we use it in postman now?

**Comment by darrenjennings** [comment]
@chakpongchung this lib has been forked and maintained separately by postman, so you would need to request on their repo.

**Pull request merged**